freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/salt/update/SaltUpdateTriggerEvent.java

```diff
@@ -9,16 +9,26 @@
 
     private String operationId;
 
+    private final boolean chained;
+
+    private final boolean finalChain;
+
     public SaltUpdateTriggerEvent(Long stackId) {
```

Comment:
This constructor still shows up as unused in my IDEA. Could you please check again if it is used? I could not find any reference.